src/controllers/MessageController.ts

```diff
@@ -0,0 +1,48 @@
+import MessageService from "../db/service/MessageService";
+import { Request, Response } from "express";
+import mongoose from "mongoose";
+import ConversationService from "../db/service/ConversationService";
+import { messages } from "../db/messages";
+
+class MessageController {
+	private messageService: MessageService;
+	private conversationService: ConversationService;
+
+	constructor() {
+		this.messageService = new MessageService();
+		this.conversationService = new ConversationService();
+	}
+
+	async sendMessage(req: Request, res: Response) {
+		const conversationId = new mongoose.Types.ObjectId(req.body.conversationId);
+		const conversation = await this.conversationService.findConversation({ _id: conversationId });
+		if (conversation?._id) {
+			let message = await this.messageService.createMessage(
+				req.body.sender,
+				req.body.message,
+				conversation._id
+			);
```

Comment:
will the database fail to create a message here due to a bad input?

Reply:
added validate in route